src/main/resources/application.yaml

```diff
@@ -34,6 +34,20 @@ spring:
           lob:
             # silence the 'wall-of-text' - unnecessary exception throw about blob types
             non_contextual_creation: true
+  mail:
+    host: ${SMTP_HOST:false}
+    password: password
```

Comment:
do we want this default value?

Reply:
any reason to not use explicit env var here?

Reply:
all env vars come in next pr when feature is actually enabled. all this is default/test setup

Reply:
I think it's best for the app to crash if it doesn't receive the host. Otherwise we might have problems figuring out what's wrong if, for example, this env var is not set.